projects/angular/src/forms/styles/_properties.forms.scss

```
@@ -65,6 +65,10 @@
           forms-variables.$clr-forms-invalid-color,
           $lightness: 30%
         )};
+      --clr-forms-textarea-padding: forms-variables.$clr-forms-baseline +
+        variables.$clr_baselineRem_2px
+        forms-variables.$clr-forms-baseline *
+        2;
```

```suggestion
      --clr-forms-textarea-padding: #{forms-variables.$clr-forms-baseline} +
        #{variables.$clr_baselineRem_2px}
        #{forms-variables.$clr-forms-baseline} *
        2;
```

Consider my suggestion.